cli/command/context/remove_test.go

```diff
@@ -18,7 +17,7 @@ func TestRemove(t *testing.T) {
 	_, err := cli.ContextStore().GetMetadata("current")
 	assert.NilError(t, err)
 	_, err = cli.ContextStore().GetMetadata("other")
-	assert.Check(t, is.ErrorType(err, errdefs.IsNotFound))
+	assert.Check(t, errdefs.IsNotFound(err))
```

Comment:
To keep the current (more informative) output of `is.ErrorType`, we should probably use the 3rd argument of `assert.Check` to write a custom error message (`wanted %T, actual %T` or something along those lines), but it will definitely make the code a lot more lengthy/verbose 😞

Reply:
Ahh, yeah I just saw you mention the nicer output (I definitely wondered why we were using `Errortype(err, errdefs.IsXxx)`) in the thread with the gotest.tools folks.

Reply:
If we want to continue using the `is.ErrorType`, I think Daniel was open to un-deprecating it (IIUC, reason for deprecating was mostly cleaning up ("stdlib now has utilities for it, and the alternative can work, so let's deprecate"). We can open a PR to do so, and see if they're ok with it.